xclim/core/calendar.py

```diff
@@ -1,31 +1,33 @@
 # -*- coding: utf-8 -*-
 """
+Calendar handling utility module.
+
 Calendar handling utilities
 ===========================
```

Comment:
Are these two lines added automatically or by hand? Because they duplicate the titles and stay there in the docs
![Capture d’écran de 2020-07-10 17-37-55](https://user-images.githubusercontent.com/20629530/87204914-205c5a00-c2d4-11ea-9641-3b2e6a872a59.png)

Reply:
I was afraid that might be the result. We may need to set an exception for module-level docstrings, otherwise we get lots of errors due to the first line not ending with a period and/or not having a blank line below it. I still have to tackle the `indices` functions so will look into this next week.

Reply:
To answer your question, changes were made by hand. If something looks off, I can't pass the buck to the linter (unfortunately).